clang/test/CXX/drs/cwg1xx.cpp

```diff
@@ -628,10 +628,12 @@ namespace cwg141 { // cwg141: 3.1
       // expected-error@-1 {{use 'template' keyword to treat 'f' as a dependent template name}}
     }
     void h() {
-      (void)t.S<int>::n; // ok
+      (void)t.S<int>::n;
```

Comment:
Previously this was considered well-formed, but now we're issuing a diagnostic. Can you elaborate?

Reply:
Makes sense, thank you.